views/partials/banners/download_firefox_banner.hbs

```diff
@@ -7,7 +7,7 @@
       </div>
     </div>
     <div class="third">
-      <a href="https://www.mozilla.org/firefox/new/?utm_campaign=ffxmdownloads" class="button" target="_blank" rel="noopener">Download Firefox</a>
+      <a href="https://www.mozilla.org/firefox/new/?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=download_banner" class="button" target="_blank" rel="noopener">Download Firefox</a>
```

Comment:
And just to round out my stupidity, do you want underscores (consistent w/ "email_footer" above) or hyphens (consistent w/ "footer-link" above):

`&utm_content=download_banner` versus "download-banner" 

per:

`&content=email_footer`
`&utm_content=footer-link`